nomenclature/region_mapping_models.py

```diff
@@ -25,28 +26,31 @@ class CommonRegion(BaseModel):
 
 class RegionAggregationMapping(BaseModel):
     model: str
+    file: FilePath
     native_regions: Optional[List[NativeRegion]]
     common_regions: Optional[List[CommonRegion]]
 
     @validator("native_regions")
-    def validate_native_regions(cls, v):
+    def validate_native_regions(cls, v, values):
         target_names = [nr.target_native_region for nr in v]
         duplicates = [
             item for item, count in Counter(target_names).items() if count > 1
         ]
         if duplicates:
             raise ValueError(
-                f"Two or more native regions share the same name: {duplicates}"
+                f"Two or more native regions share the same name: {duplicates} in "
```

Comment:
I think that the three ValueErrors are sufficiently similar so that moving the actual raising of the error to a utility method like `_raise_error(msg, values)` would improve the code.

Reply:
In particular because of the addition `.relative_to(Path.cwd())`...

Reply:
Ah yes, good point but I think I'd rather create a custom error, something along the lines of `NameOverlapError`.

Reply:
Nice!